Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/pdf/ui/WebView.kt

```diff
@@ -0,0 +1,23 @@
+package de.rki.coronawarnapp.covidcertificate.pdf.ui
+
+import android.webkit.WebSettings
+import android.webkit.WebView
+import android.webkit.WebViewClient
+
+fun WebView.setupWebView(onPageFinished: (view: WebView) -> Unit) {
```

Comment:
I would name this more specific, because this cannot be applied to any webview

Reply:
Let's change this in another PR so we can get @mtwalli PR in now? We have the approval from UA, pushing some changes now would dismiss all approvals and we'd have to wait some more.

Reply:
@schauersbergern if you want, rename it the name you like ?